cmake/FesapiJavaExample.java

```diff
@@ -24,43 +24,36 @@ Licensed to the Apache Software Foundation (ASF) under one
-import com.f2i.energisticsStandardsApi.witsml1__LengthUom;
-import com.f2i.energisticsStandardsApi.witsml1__WellVerticalCoordinateUom;
-import com.f2i.energisticsStandardsApi.common.AbstractHdfProxy;
-import com.f2i.energisticsStandardsApi.common.EpcDocument;
-import com.f2i.energisticsStandardsApi.resqml2.PropertyKind;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.ContinuousProperty;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.FaultInterpretation;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.GenericFeatureInterpretation;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.Grid2dRepresentation;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.Horizon;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.HorizonInterpretation;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.LocalDepth3dCrs;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.PointSetRepresentation;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.PolylineRepresentation;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.SeismicLatticeFeature;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.SeismicLineFeature;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.SeismicLineSetFeature;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.TectonicBoundaryFeature;
-import com.f2i.energisticsStandardsApi.resqml2_0_1.TriangulatedSetRepresentation;
-import com.f2i.energisticsStandardsApi.witsml1_4_1_1.CoordinateReferenceSystem;
-import com.f2i.energisticsStandardsApi.witsml1_4_1_1.Well;
+import com.f2i.energisticsStandardsApi.${FESAPI_COMMON_NS}.AbstractHdfProxy;
+import com.f2i.energisticsStandardsApi.${FESAPI_COMMON_NS}.EpcDocument;
+import com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_NS}.PropertyKind;
+import com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_0_1_NS}.ContinuousProperty;
+import com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_0_1_NS}.GenericFeatureInterpretation;
+import com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_0_1_NS}.Grid2dRepresentation;
+import com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_0_1_NS}.Horizon;
+import com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_0_1_NS}.HorizonInterpretation;
+import com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_0_1_NS}.LocalDepth3dCrs;
+import com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_0_1_NS}.PolylineRepresentation;
+import com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_0_1_NS}.SeismicLatticeFeature;
+import com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_0_1_NS}.SeismicLineFeature;
+import com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_0_1_NS}.SeismicLineSetFeature;
+import com.f2i.energisticsStandardsApi.${FESAPI_RESQML2_0_1_NS}.TriangulatedSetRepresentation;
+import com.f2i.energisticsStandardsApi.${FESAPI_WITSML2_0_NS}.Well;
 
 public class FesapiJavaExample {
 	/**
 	 * Loading the FesapiCpp native library
 	 */
 	static {
 		try {
-			System.loadLibrary("${CPP_LIBRARY_NAME}.${Fesapi_VERSION}");
+			System.loadLibrary("FesapiCppUnderDev.0.15.0.0");
```

Comment:
I don't think it should be hard coded. The previous way looks to me better.

Reply:
Fixed.